### ISSUE TITLE
Does not show current config

### ISSUE THREAD

**Issue by Akyshnik** [daedreth/ihsec#6]
On my machine, ihsec wouldn't show current config for some reason. Hopefully, I identified the problem that caused it. There were some redundant slashes in ihsec and asterisks that prevented identification of configuration that was currently in use on my computer. Not sure whether it wasn't working correctly only on my machine. Please take a look at my modified version Akyshnik/ihsec@b9ee9c5d5e78d87bd5b970d803e2d96f8a78e274

**Comment by Akyshnik** [comment]
I didn't submit pull request as I am not sure whether this issue is present only on my computer.

**Comment by daedreth** [comment]
Could you show me a screenshot of the "list" output? You may have similar names in your config list and this is what trips up ihsec.
Just a theory though, if it's confirmed I'd totally fix that.

**Comment by Akyshnik** [comment]
Sure! On the screenshot below I run "list" and "set" and then "list" again. As you can see, nothing has changed. 

Furthermore, I wrote a simple script to show the contents of the variables `ihsec.sh` uses, script is provided below. You can see that there `$CONFIGDIRS` have some extra slashes, while `$CURR_CONFIG` hasn't. That prevents ihsec to identify config that is currently in use.

```bash
#!/bin/bash

# These variables are defined exactly as they are in ihsec.sh
CONFIGS_DIR=$HOME'/.ihsec/'
CONFIGDIRS="$CONFIGS_DIR/*/"
CURR_CONFIG=$(readlink -f $HOME/.emacs.d)

echo CONFIGS_DIR $CONFIGS_DIR
echo CONFIGDIRS $CONFIGDIRS
echo CURR_CONFIG $CURR_CONFIG

echo
echo Dirs in CONFIGDIRS:
for DIR in $CONFIGDIRS; do
    echo $DIR
    if [[ $DIR == *"$CURR_CONFIG"* ]]; then echo "current config"; else echo "other config";
    fi
    echo
done
```
![Screenshot](https://user-images.githubusercontent.com/23380632/38821408-78947492-41b1-11e8-8804-2a72faf1b04f.png)

**Comment by Akyshnik** [comment]
One solution would be to add one slash to the `CURR_CONFIG` variable, the other one would be to delete extra ones from `CONFIGDIRS`. In the commit I referenced in the original post, I deleted the extra ones.

**Comment by Akyshnik** [comment]
Actually, forget the commit that I referenced before. It turned out, the whole thing can be fixed for me by deleting only one slash. Here it is: Akyshnik/ihsec@4457ff52fb330a4ffc31f9cdcffe06b8f1ca2c29

**Comment by Akyshnik** [comment]
And here is the screenshot of the output of my script. There is no redundant slashes anymore and everything just werkz.

But it is very confusing though — if I had this issue then everybody should have it.

![Screenshot](https://user-images.githubusercontent.com/23380632/38822726-5e5eaa62-41b5-11e8-8578-dabfd1db8d1c.png)

**Comment by daedreth** [comment]
awesome, I sadly have no ability to check it out right now, but I'll have a look tomorrow and if everything works on my end, I'll request a PR :°)

**Comment by a-hausmann** [comment]
I concur with Akyshnik, though my fix is different.  I still removed a slash, but it was trailing slash in definition of CONFIGS_DIR. No need for trailin slash after ".ihsec" as that is taken care of in CONFIGDIRS definition, and leaving CONFIGDIRS as is makes it much more readable.

To Akyshnik's point, I also had the same problem, script works fine but didn't show current configuration, until I fixed the shell script.  Am am running Ubuntu 18.04, standard bash.

**Comment by daedreth** [comment]
Fixed in f6e185a927d70839d3a3e84bb493d24d09452271
I hope.
